algorithmic_efficiency/workloads/criteo1tb/criteo1tb_jax/workload.py

```diff
@@ -165,37 +27,39 @@ def model_params_types(self):
       self._param_types = param_utils.jax_param_types(self._param_shapes)
     return self._param_types
 
-  def output_activation_fn(self,
-                           logits_batch: spec.Tensor,
-                           loss_type: spec.LossType) -> spec.Tensor:
-    """Return the final activations of the model."""
-    pass
-
   def loss_fn(
       self,
       label_batch: spec.Tensor,  # Dense (not one-hot) labels.
       logits_batch: spec.Tensor,
-      label_smoothing: float = 0.0,
-      mask_batch: Optional[spec.Tensor] = None) -> spec.Tensor:
-    smoothed_targets = optax.smooth_labels(label_batch, label_smoothing)
+      mask_batch: Optional[spec.Tensor] = None,
+      label_smoothing: float = 0.0) -> spec.Tensor:
+    del label_smoothing
```

Comment:
Do we not want to be able to tune label smoothing for all workloads?

Reply:
We do, I just removed this here because for Criteo it's only binary classification, so I figured it would not be useful/valid to tune this, but I may have been mistaken? I added it to the agenda to chat about on Thurs to see if we wanna keep this.